docs/jupyter-in-polyglot-notebooks.md

```diff
@@ -6,15 +6,22 @@ This feature is currently in preview.
 
 ## Setup
 Before you begin, make sure you have the following installed:
-1. [The Anaconda distribution](https://docs.anaconda.com/free/anaconda/install/index.html)
-2. [Python](https://www.python.org/downloads/) and/or [R](https://cran.r-project.org/)
+1. [The Anaconda distribution](https://docs.anaconda.com/free/anaconda/install/index.html). Comes with Python and Jupyter.
+2. OR Install [Python](https://www.python.org/downloads/) and add to your PATH. You would need to install [Jupyter](https://jupyter.org/install#jupyter-notebook)
+3. If you are working with R - Install [R](https://cran.r-project.org/)
 
 ## Connecting to a Python kernel
 Run the following command in a notebook cell:
+If working with Anaconda:
 ```
 #!connect jupyter --kernel-name pythonkernel --conda-env base --kernel-spec python3
 ```
 
+If working with Python and Jupyter directly:
```

Comment:
What does it mean to be working with Python and Jupyter directly as opposed to with Anaconda?

Reply:
Line 2 above in Setup. You can install Python on your machine and add it to Path and then pip install Jupyter.

Reply:
Got it. can we change the verbiage to;

"If working with Jupyter using Anaconda"

and:

"If you are working with Python and Jupyter directly without Anaconda" 

to make it more explicit

Reply:
Done!